lib/grack.rb

```diff
@@ -1,5 +1,10 @@
 require "grack/bundle"
-
+require 'zlib'
+require 'rack/request'
+require 'rack/response'
+require 'rack/utils'
+require 'time'
+require 'grack/git'
```

Comment:
Your right, there are no dups; late night commit. I thought some of the rack ones were dups but they aren't.